### PR TITLE
Add bond mac address to nmtui

### PR DESCRIPTION
**Pull request by mar-kolya**
Also fix bug in nmtui vlan and wireless pages when configuration wasn't saved when connection was created.

### CODE REVIEW

File: clients/tui/nmt-page-vlan.c [hunk 1]
```diff
@@ -35,7 +35,6 @@ G_DEFINE_TYPE (NmtPageVlan, nmt_page_vlan, NMT_TYPE_EDITOR_PAGE_DEVICE)
 #define NMT_PAGE_VLAN_GET_PRIVATE(o) (G_TYPE_INSTANCE_GET_PRIVATE ((o), NMT_TYPE_PAGE_VLAN, NmtPageVlanPrivate))
 
 typedef struct {
-	NMSettingWired *s_wired;
 
 } NmtPageVlanPrivate;
```
Comment (bengal) [inline clients/tui/nmt-page-vlan.c]: An empty private structure will raise warnings from GLib at runtime. You should get rid of it altogether and remove the g_type_class_add_private().

File: clients/tui/nmt-page-wifi.c [hunk 2]
```diff
@@ -41,7 +41,6 @@ G_DEFINE_TYPE (NmtPageWifi, nmt_page_wifi, NMT_TYPE_EDITOR_PAGE_DEVICE)
 #define NMT_PAGE_WIFI_GET_PRIVATE(o) (G_TYPE_INSTANCE_GET_PRIVATE ((o), NMT_TYPE_PAGE_WIFI, NmtPageWifiPrivate))
 
 typedef struct {
-	NMSettingWirelessSecurity *s_wsec;
 
 } NmtPageWifiPrivate;
```
Comment (bengal) [inline clients/tui/nmt-page-wifi.c]: clients/tui/nmt-page-wifi.c:179:22: error: unused variable ‘priv’ [-Werror=unused-variable]
  NmtPageWifiPrivate *priv = NMT_PAGE_WIFI_GET_PRIVATE (object);

Comment (bengal) [inline clients/tui/nmt-page-wifi.c]: The rest LGTM!

**Comment by mar-kolya** [comment]
@bengal thanks for your response!

I've pushed an updated version of these patches.
Thanks!

**Comment by thom311** [comment]
v2 looks good to me too.

Merged as 53c0e5fa977f6c29421b77fdd300db9707ad5682

Nice work, Nikolay :)

**Pull request closed without merging**